--- a/src/SerialTty.cpp
+++ b/src/SerialTty.cpp
@@ -41,6 +41,9 @@
 #include "common.hpp"
 #include "DeviceSettings.hpp"
 
+#ifndef IUCLC
+# define IUCLC 0
+#endif
 
 namespace fs = boost::filesystem;
 using namespace std;
--- a/src/gant/antlib.c
+++ b/src/gant/antlib.c
@@ -20,6 +20,10 @@
 #define MAXCHAN 32
 #define BSIZE 8*10000
 
+#ifndef IUCLC
+# define IUCLC 0
+#endif
+
 #define uchar unsigned char
 
 #define hexval(c) ((c >= '0' && c <= '9') ? (c-'0') : ((c&0xdf)-'A'+10))
